foreign.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const path = require('path');
+const foreign = require('foreign');
+const readFile = (filename, callback) => fs.readFile(filename, {encoding: 'utf8'}, callback);
+const relative = (filename) => path.join(process.argv[2], filename);
+
+const main = () => {
+  const complete = (error, files) => {
+    error ?
+      process.stderr.write(error.message + '\n') :
+      process.stdout.write(files.join(''));
+
+    process.exit(error ? 1 : 0);
+  };
+
+  readFile(relative('index.txt'), (error, index) => {
+    if(error) return complete(error);
+
+    foreign.parallel(readFile, index.match(/^.*(?=\n)/gm).map(relative), complete);
+  });
+}
+
+if (process.argv[1] === __filename) main();
```

Comment:
[Newline at end of file][1], please. :)


[1]: https://github.com/editorconfig/editorconfig/wiki/Newline-at-End-of-File-Support